tailcfg/tailcfg.go

```diff
@@ -1221,11 +1222,22 @@ type CapGrant struct {
 	CapMap PeerCapMap `json:",omitempty"`
 }
 
-// PeerCapability is a capability granted to a node by a FilterRule.
-// It's a string, but its meaning is application-defined.
-// It must be a URL, like "https://tailscale.com/cap/file-sharing-target" or
-// "https://example.com/cap/read-access".
-type PeerCapability string
+// CapabilityName is a string used to identify a specific capability. Its
+// meaning is application-defined.
+//
+// It must be a URL like "https://tailscale.com/cap/file-send", or a well-known
+// capability name like 'funnel'.
+type CapabilityName string
```

Comment:
Is there any reason for this? why not just make the other two be of type string. there's no inheritance going on here.

what would use this type?

Reply:
this was only for documentation purposes, can remove